portality/static/js/formulaic.js

```diff
@@ -1930,6 +1931,8 @@ var formulaic = {
 
                 let selector = "[name='" + this.fieldDef.name + "']";
 
+                $(selector).on("focus", formulaic.widgets._select2_shift_focus);
```

Comment:
We will need to do all the other select2-based widgets too: `Select`, `TagEntry`, `Autocomplete`

Reply:
They are added to the code, just for some reason not shown in this diff. Super weird:

Autocomplete: https://github.com/DOAJ/doaj/blob/0580a5541ea039da5960b0b84dd1cbb0b68b09fe/portality/static/js/formulaic.js#L1934
Select: https://github.com/DOAJ/doaj/blob/0580a5541ea039da5960b0b84dd1cbb0b68b09fe/portality/static/js/formulaic.js#L1736
Taglist: https://github.com/DOAJ/doaj/blob/0580a5541ea039da5960b0b84dd1cbb0b68b09fe/portality/static/js/formulaic.js#L1802
Tag Entry: https://github.com/DOAJ/doaj/blob/0580a5541ea039da5960b0b84dd1cbb0b68b09fe/portality/static/js/formulaic.js#L1827